src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -36,6 +36,7 @@
 using ResourceNames = Dynamo.Wpf.Interfaces.ResourceNames;
```

Comment:
This is the new image capture logic that tries to determine the bounds of `DragCanvas` through its descendants, then use that as the image size before capturing the entire sub visual tree.